packages/flutter/lib/src/widgets/will_pop_scope.dart

```diff
@@ -11,6 +11,81 @@ import 'routes.dart';
 /// Registers a callback to veto attempts by the user to dismiss the enclosing
 /// [ModalRoute].
 ///
+/// WillPopScope prevents gestures that makes current route pop such as backswipe gesture on iOS. 
+/// To enable this behaviour, use custom route which overrides `hasScopedWillPopCallback` 
+/// getter and returns false on either custom `MaterialPageRoute` or `CupertinoPageRoute`.
+///
+/// {@tool dartpad --template=freeform}
+///
+/// This sample demonstrates how to implement WillPopScope and prevent user from popping
+/// current route.
+///
+/// ```dart
+/// /// Flutter code sample for WillPopScope
+/// /// This sample demonstrates how to register a callback for when the user wants
+/// /// to dismiss the current [Route].
+///
+/// /// `onWillPop` callback can be used to show warning to the user on popping
+/// /// the current [Route], preventing user from going to back to unintended Routes.
```

Comment:
remove extra `to`. `going back to unintended Routes`. Not sure if last word should `Routes` or `routes`